staging/src/k8s.io/apiserver/pkg/registry/rest/delete.go

```diff
@@ -39,6 +39,7 @@ type GarbageCollectionPolicy string
 const (
 	DeleteDependents GarbageCollectionPolicy = "DeleteDependents"
 	OrphanDependents GarbageCollectionPolicy = "OrphanDependents"
+	Unsupported      GarbageCollectionPolicy = "Unsupported"
```

Comment:
> With this implementation, the Unsupported resource won't break the background deletion chain.

It does because the two sets of ignores match.  I suppose if someone modified one and not the other they'd have a problem.  We could expose GC-ability through discovery, but that feels separate.